CHANGELOG.md

```diff
@@ -17,6 +17,17 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.22.0] - 2024-02-21
+### Added
+- Data point subscriptions reaches General Availability (GA).
+  - Use the new [Data point subscriptions](https://developer.cognite.com/dev/concepts/data_point_subscriptions/)
+    feature to configure a subscription to listen to changes in one or more
+    time series (in ingestion order).
+    The feature is intended to be used where data points consumers need to keep up to date with
+    changes to one or more time series without the need to read the entire time series again.
+### Changed
+- Removed the `ignore_unknown_ids` flag from `client.time_series.subscriptions.retrieve()` to stay consistent with other resource types.
```

Comment:
@doctrino I thought that all the `retrieve_multiple` functions had `ignore_unknown_ids` so that the users can choose, but the `retrieve` functions always returns None if it is not found?

Reply:
It is only the new `units` resource that accepts `ignore_unknown_ids` on singular `retrieve` functions. Everything else returns the resource or None.

![image](https://github.com/cognitedata/cognite-sdk-python/assets/819391/e74e7ef0-799d-4221-9940-15fca24de46d)

Reply:
yeah, it's always redundant on `retrieve`, but technically a breaking change to remove the parameter nonetheless

Reply:
ah, but we're moving from alpha so it's fine to remove.

Reply:
I think we should do it carefully even with alpha. Two sec, need to look up how we have done it before.

Reply:
Maybe off-topic, but why are you moving away from `retrieve_multiple`?

Reply:
I vote to keep `ignore_unknown_ids`, unless we are adding a `retrieve_multiple` method.

Reply:
Then I think it would be better to add `retrieve_multiple` and let the `retrieve` method be consistent with the rest of the API.

Reply:
I just learned from @erlendvollset that this only accepts a single external ID and so now I fully agree with the change (to remove it)